crypto/x509/x509_test.cc

```diff
@@ -4156,3 +4156,93 @@ TEST(X509Test, TrustedFirst) {
                X509_VERIFY_PARAM_clear_flags(param, X509_V_FLAG_TRUSTED_FIRST);
              }));
 }
+
+TEST(X509Test, Policy) {
```

Comment:
Some of these are duplicating tests from https://github.com/aws/aws-lc/pull/905.

See [test fixture](https://github.com/aws/aws-lc/blob/fips-2021-10-20/crypto/x509/x509_test.cc#L1789) `X509Test.PolicyTreeBasic`.